extensions/wikia/DesignSystem/services/DesignSystemGlobalNavigationWallNotificationsService.class.php

```diff
@@ -24,14 +24,18 @@ public function Notification() {
 
 		$authors = [ ];
 		foreach ( $notify[ 'grouped' ] as $notify_entity ) {
-			$authors[] = [
-				'displayname' => $notify_entity->data->msg_author_displayname,
-				'username' => $notify_entity->data->msg_author_username,
-				'avatar' => AvatarService::renderAvatar(
-					$firstEntity->data->msg_author_username,
-					AvatarService::AVATAR_SIZE_SMALL_PLUS
-				)
-			];
+			if (!empty($notify_entity->data)) {
```

Comment:
Minor [our PHP guidelines](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md) are a superset of [MWs PHP guidelines](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP) which calls for space around parentheses like these.